### ISSUE TITLE
Dynamic prefix issue

### ISSUE THREAD

**Issue by ilearnbydoing** [bramus/router#168]
This is not working 
/{slug}/volume-{volume_id}/issue-{issue_id}

while this work

/{slug}/volume/{volume_id}/issue/{issue_id}

Is dynamic URL Prefix not supported?

**Comment by uvulpos** [comment]
No, not implemented yet, but you are welcome to contribute it 🙂 
Hint: https://github.com/bramus/router/blob/master/src/Bramus/Router/Router.php#L353
